app/screens/RestaurantDetail/ReviewList.jsx

```diff
@@ -0,0 +1,25 @@
+import { React, useState, useEffect } from 'react';
+import axios from 'axios';
+import { Text, View, Card } from 'react-native';
+import ReviewEntry from './ReviewEntry';
+
+function ReviewList() {
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    axios.get('http://localhost:8080/api/v1/reviews')
```

Comment:
You may not need to do a request to reviews once my PR #13 is merged. Restaurants will include reviews. Otherwise, if you need to fetch reviews for a restaurant, you need to pass in a restaurantId query parameter.

Reply:
sure. We were using this request for development only. Will update it accordingly.